server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -86,9 +89,19 @@ public long getTimeInMillis() {
 
         @Override
         public void writeTo(StreamOutput out) throws IOException {
-            out.writeVLong(current);
+            if (current < 0) {
+                logger.debug("[PhaseStatsLongHolder] 'current' is negative: {}", current);
+                out.writeVLong(0);
+            } else {
+                out.writeVLong(current);
+            }
```

Comment:
This doesn't look like the right way to handle it since ideally current should never be < 0.

Reply:
This is not a permanent solution but will prevent negative values from causing `failed_node_exception`

Reply:
Can we use writeZLong for now?
I am worried that by doing this, we will lose track of any issues occurring due to these stats.
By using writeZLong, we will atleast avoid ser/der issues, and still continue to debug and fix the actual issue.

Reply:
Changed to ZLong and removed checks in SearchRequestContext.